pkg/operator/queues.go

```diff
@@ -446,7 +446,7 @@ func (op *Reporting) handleErr(logger log.FieldLogger, err error, objType string
 
 	// This controller retries up to maxRequeues times if something goes wrong.
 	// After that, it stops trying.
-	if queue.NumRequeues(obj) < maxRequeues {
+	if maxRequeues == -1 || queue.NumRequeues(obj) < maxRequeues {
```

Comment:
Is there a point to the right side of this conditional? Seems like it's always going to evaluate to true.

Reply:
Yep, maxRequeues is normally something like 5 or 10, but in this case we're setting it to -1.